server/schedulers/balance_test.go

```diff
@@ -935,6 +935,7 @@ var _ = Suite(&testBalanceHotWriteRegionSchedulerSuite{})
 type testBalanceHotWriteRegionSchedulerSuite struct{}
 
 func (s *testBalanceHotWriteRegionSchedulerSuite) TestBalance(c *C) {
+	statistics.Simulating = true
```

Comment:
`Simulating` is only used for simulator, I don't think it is proper to use it here.